nvdaHelper/vbufBase/storage.cpp

```diff
@@ -321,6 +321,17 @@ bool VBufStorage_fieldNode_t::addAttribute(const std::wstring& name, const std::
 	return true;
 }
 
+bool VBufStorage_fieldNode_t::getAttribute(const std::wstring& name, std::wstring* value) {
+	LOG_DEBUG(L"Getting attribute " << name);
+	VBufStorage_attributeMap_t::const_iterator foundAttrib = attributes.find(name);

```

Comment:
You can just do `auto foundItr = attributes.find(name);`
If you mark the function as `const`, then `attributes` will be `const` and return a `const` iterator.